src/frequenz/client/microgrid/_component_data.py

```diff
@@ -70,18 +70,30 @@ class MeterData(ComponentData):
     """A wrapper class for holding meter data."""
 
     active_power: float
-    """The 3-phase active power, in Watts, represented in the passive sign convention.
-            +ve current means consumption, away from the grid.
-            -ve current means supply into the grid.
+    """The 3-phase active power, in W (Watt).
```

Comment:
Maybe it's better to be explicit here?
```suggestion
    """The total 3-phase active power, in W (Watt).
```